tests/bluechi_test/machine.py

```diff
@@ -183,6 +210,17 @@ def gather_coverage(self, data_coverage_dir: str) -> None:
 
         self.client.get_file(f"{coverage_file}", data_coverage_dir)
 
+    def cleanup_valgrind_logs(self):
+        self.client.exec_run(f"rm -f {BluechiMachine.valgrind_log_path_controller}")
+        self.client.exec_run(f"rm -f {BluechiMachine.valgrind_log_path_agent}")
+
+    def cleanup_journal_logs(self):
+        self.client.exec_run("journalctl --flush --rotate")
+        self.client.exec_run("journalctl --vacuum-time=1s")
```

Comment:
By using [--since](https://www.freedesktop.org/software/systemd/man/latest/journalctl.html#-S) this should be possible, yes. Although we probably want to get the timestamp from the individual machines since they could use different times than the executor machine - low chance, but possible.
I think clearing the journal logs is more clearer and straight-forward, but we can discuss this and change it in a followup PR.